pwned/models.py

```diff
@@ -0,0 +1,12 @@
+from django.contrib.auth import get_user_model
+from django.db import models
+
+
+UserModel = get_user_model()
+
+
+class PwnedRecord(models.Model):
```

Comment:
I'm not sure that this app should be providing this model at all, especially with fields (`notified`) that it doesn't itself use. To me this seems like a good candidate for a signal instead, which projects can then do what they like with - and would allow a much wider set up use cases without creating a potentially unused table in the database.